spec/examples/ex4-overflow.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<tt xmlns="http://www.w3.org/ns/ttml" xmlns:tts="http://www.w3.org/ns/ttml#styling" xml:lang="en" tts:extent="320px 240px">
+  <head>
+    <styling>
+      <style xml:id="s1" tts:backgroundColor="black" tts:wrapOption="noWrap" tts:padding="6px"/>
+      <style xml:id="s2" tts:color="red" tts:backgroundColor="purple" tts:fontFamily="proportionalSansSerif" tts:fontSize="10px"/>
```

Comment:
I'm finding the colour contrast between the purple and the red really horrible (and the luminance contrast is very low too) - please could you change one of them, e.g. change the purple to grey?

Reply:
Grey does not provide sufficient contrast with the background. Please pick another color.

Reply:
White?